src/server/chromium/crPage.ts

```diff
@@ -708,7 +723,9 @@ class FrameSession {
   }
 
   async _onBindingCalled(event: Protocol.Runtime.bindingCalledPayload) {
-    const context = this._contextIdToContext.get(event.executionContextId)!;
+    const context = this._contextIdToContext.get(event.executionContextId);
+    if (!context)
+      return;
     const pageOrError = await this._crPage.pageOrError();
     if (!(pageOrError instanceof Error))
       await this._page._onBindingCalled(event.payload, context);
```

Comment:
onDialog: let's check that frameManager has *us* in a frame tree, and in this case report it.